server/coordinator_test.go

```diff
@@ -795,32 +795,42 @@ func (s *testOperatorControllerSuite) TestStoreOverloaded(c *C) {
 	cfg, opt, err := newTestScheduleConfig()
 	c.Assert(cfg, NotNil)
 	c.Assert(err, IsNil)
-	cfg.StoreBalanceRate = 10
 	tc := newTestCluster(opt)
 	hbStreams, cleanup := getHeartBeatStreams(c, tc)
 	defer cleanup()
 	defer hbStreams.Close()
 	oc := schedule.NewOperatorController(tc.RaftCluster, hbStreams)
 	lb, err := schedule.CreateScheduler("balance-region", oc)
 	c.Assert(err, IsNil)
-
-	c.Assert(tc.addRegionStore(4, 40), IsNil)
-	c.Assert(tc.addRegionStore(3, 40), IsNil)
-	c.Assert(tc.addRegionStore(2, 40), IsNil)
+	// scheduling one time needs 60 seconds
+	// and thus it's large enough to make sure that only schedule one time
+	cfg.StoreBalanceRate = 1
+	c.Assert(tc.addRegionStore(4, 100), IsNil)
+	c.Assert(tc.addRegionStore(3, 100), IsNil)
+	c.Assert(tc.addRegionStore(2, 100), IsNil)
 	c.Assert(tc.addRegionStore(1, 10), IsNil)
 	c.Assert(tc.addLeaderRegion(1, 2, 3, 4), IsNil)
-	start := time.Now()
+	region := tc.GetRegion(1).Clone(core.SetApproximateSize(60))
+	tc.putRegion(region)
 	op1 := lb.Schedule(tc)[0]
 	c.Assert(op1, NotNil)
 	c.Assert(oc.AddOperator(op1), IsTrue)
+	c.Assert(oc.RemoveOperator(op1), IsTrue)
```

Comment:
why do we need to remove operator

Reply:
It will occupy the position with the same region ID.